nnvm/python/nnvm/frontend/mxnet.py

```diff
@@ -263,6 +263,12 @@ def _expand_dims(inputs, attrs):
     new_attrs['axis'] = _required_attr(attrs, 'axis')
     return _get_nnvm_op(op_name)(*inputs, **new_attrs)
 
+def _l2_normalize(inputs, attrs):
+    op_name, new_attrs = 'l2_normalize', {}
+    mode = attrs.get('mode', 'instance')
+    new_attrs['axis'] = 0 if mode == 'instance' else 1
```

Comment:
L2 normalize also support for spatial axis. Can consider 'spatial' case also

Reply:
@PariksheetPinjari909  Ok. i already fixed this